From 00cba650c86f494b4a5905d89d304fef2380b0b7 Mon Sep 17 00:00:00 2001
From: Tom Rix <trix@redhat.com>
Date: Mon, 3 Apr 2023 20:45:01 -0400
Subject: [PATCH backport 6.1.42 2/2] pinctrl: mlxbf3: set varaiable
 mlxbf3_pmx_funcs storage-class-specifier to static

smatch reports
drivers/pinctrl/pinctrl-mlxbf3.c:162:20: warning: symbol
  'mlxbf3_pmx_funcs' was not declared. Should it be static?

This variable is only used in one file so it should be static.

Signed-off-by: Tom Rix <trix@redhat.com>
Link: https://lore.kernel.org/r/20230404004501.1913144-1-trix@redhat.com
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
---
 drivers/pinctrl/pinctrl-mlxbf3.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pinctrl/pinctrl-mlxbf3.c b/drivers/pinctrl/pinctrl-mlxbf3.c
index 3698f7bbd88d..d9944e6a0af9 100644
--- a/drivers/pinctrl/pinctrl-mlxbf3.c
+++ b/drivers/pinctrl/pinctrl-mlxbf3.c
@@ -159,7 +159,7 @@ static const struct pinctrl_ops mlxbf3_pinctrl_group_ops = {
 static const char * const mlxbf3_gpiofunc_group_names[] = { "swctrl" };
 static const char * const mlxbf3_hwfunc_group_names[]   = { "hwctrl" };
 
-struct pinfunction mlxbf3_pmx_funcs[] = {
+static struct pinfunction mlxbf3_pmx_funcs[] = {
 	PINCTRL_PINFUNCTION("hwfunc", mlxbf3_hwfunc_group_names, 1),
 	PINCTRL_PINFUNCTION("gpiofunc", mlxbf3_gpiofunc_group_names, 1),
 };
-- 
2.25.1

